From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dan Muey <daniel.muey@webpros.com>
Date: Thu, 22 May 2025 15:10:22 +0000
Subject: [PATCH 3/4] patch re.c for Ubuntu builds

---
 apache2/re.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/apache2/re.c b/apache2/re.c
index d067c68..bf35a3e 100644
--- a/apache2/re.c
+++ b/apache2/re.c
@@ -464,6 +464,11 @@ end:
     if (my_error_msg) {
         if (msr) msr_log(msr, 9, "%s", my_error_msg);
         else ap_log_error(APLOG_MARK, APLOG_INFO, 0, NULL, "%s", my_error_msg);
+
+        if (msr)
+            msr_log(msr, 9, "%s", my_error_msg);
+        else
+            ap_log_error(APLOG_MARK, APLOG_INFO, 0, NULL, "%s", my_error_msg);
     }
     if (target_list != NULL) free(target_list);
     if (replace != NULL) free(replace);
